Makefile

```diff
@@ -1,5 +1,5 @@
 PROJECT = mysql_poolboy
 DEPS = mysql poolboy
-dep_mysql = git https://github.com/mysql-otp/mysql-otp.git 0.8.0
+dep_mysql = git https://github.com/mysql-otp/mysql-otp.git 1.0.0
 dep_poolboy = git https://github.com/devinus/poolboy.git 1.4.2
```

Comment:
OK. What if we change the deps in the makefile and forget to run the command that generates the rebar file before commit? Then we would still have differences betweed the two types of deps... I think a testcase that parses the makefile and the rebar file and compares the deps would be more safe and not that hard to write.